context("test_error")

test_that("test_error behaves as expected", {

})

